docs/synapse/userguides/syn_tools_genpkg.ipynb

```diff
@@ -0,0 +1,242 @@
+{
+ "cells": [
+  {
+   "cell_type": "raw",
+   "metadata": {},
+   "source": [
+    ".. highlight:: none\n",
+    "\n",
+    ".. _syn-tools-genpkg:\n",
+    "\n",
+    "Synapse Tools - genpkg\n",
+    "========================\n",
+    "\n",
+    "The Synapse ``genpkg`` tool can be used to generate a Storm :ref:`gloss-package` containing new Storm commands and Storm modules from a YAML prototype and optionally push it to a Cortex or PkgRepo. \n",
```

Comment:
prototype is a weird word

Reply:
specification would be a good alternative.

Reply:
`definition` would be best.